### ISSUE TITLE
A11y_SQLAzureDataStudio_Notebook_Create new notebook_Keyboard:There is a trap present for the editors.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9405]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio; #A11yMAS; #A11ySev2; #A11yTCS; #SQL Azure Data Studio; #Benchmark; #DesktopApp; #MacOs; #Keyboard; #MAS2.1.2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS2.1.2

**Repro Steps:**
1.Launch Azure Data Studio.
2.Click File then New Notebook.
3.Click Attach To and select a SQL connection.
4.Click + Text to add a text cell. Add some text.
5.Navigate using tab inside the code cell.

**Actual:**
When enter is hit on +text button the focus moves into the editor with tab the navigation is stuck in the editor itself, thus the buttons present in the cells are not navigatable.
Same issue with the code button.

**Expected:**
There should be a key provided to move out of edit mode.The key should be specified visually on screen and on tab the button more option should be accessible.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Keyboard users will not be able to navigate out of the cell.

**MAS Reference:**
**MAS2.1.2:**https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ebdc3fe1-09a9-4aa2-a29b-e97764774e6c}

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EaR4F7UHnF1JjaqT1X-ojHwBc3rzooe8aLiD1B3fQKJB_g?e=KtI5L2

**Comment by alanrenmsft** [comment]
this is by design, you can change the tab focus mode, more information: https://code.visualstudio.com/docs/editor/accessibility#_tab-trapping

**Comment by manaswita-chougule** [comment]
As mentioned in above comment the focus can be toggled from editor window to exit keyboard tab user can follow shortcut keys documented in [https://code.visualstudio.com/docs/editor/accessibility#_tab-trapping](https://code.visualstudio.com/docs/editor/accessibility#_tab-trapping). Hence adding closed tag.
#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-MAC-Feb2020;
